lib/libzutil/zutil_import.c

```diff
@@ -2045,21 +1252,19 @@ zpool_find_import_impl(libpc_handle_t *hdl, importargs_t *iarg)
 	pthread_mutex_init(&lock, NULL);
 
 	/*
-	 * Locate pool member vdevs using libblkid or by directory scanning.
+	 * Locate pool member vdevs by blkid or by directory scanning.
```

Comment:
On FreeBSD is there an blkid equivalent which can be used here?  Or will only the scan and cache options be implemented?

Reply:
There may be a possibility of using libblkid (I see a port for it anyway) but what I anticipate will end up here on FreeBSD is the code we have to scan our GEOM tree.